src/files/XzFile.cpp

```diff
@@ -140,6 +141,8 @@ void XzFile::seek(uint64_t position) {
 
     auto count = this->read(buffer, static_cast<size_t>(position));
     assert(count == position);
+    // silent "unused variable" when compiling without assertions
+    (void)count;
```

Comment:
Maybe it would be a good idea to add a macro like `CHFL_UNUSED` which expands to this?

Reply:
We could, but I don't like having an extra global macro, and there are very few use of this pattern. Do you feel it would be worth it, at least from the code clarity point of view?